coderd/rbac/builtin.go

```diff
@@ -17,6 +17,12 @@ const (
 
 	orgAdmin  string = "organization-admin"
 	orgMember string = "organization-member"
+
+	// The below roles are for system internal use only and are
+	// not assignable to users.
+	system         string = "system"
+	systemReadOnly string = "system-read-only"
+	autostart      string = "auto-start"
```

Comment:
Remove constants, and have the role names only returnable hidden in the function call.